### PR TITLE
Matplotlib

### PR DESCRIPTION
**Pull request by kinga322**
Add entry in the menu to save file. Add show_sequences argument for dotplot.

### CODE REVIEW

**Comment by krassowski** [comment]
I would be happy to merge this, but there are several conflicts with master. Could you try to resolve them and update this PR afterwards?

**Comment by kinga322** [comment]
I resolved all conflicts

**Comment by krassowski** [comment]
When running from command line:
```bash
./dotplot.py --fasta 1.fa 2.fa
```

The program throws:
```python
Traceback (most recent call last):
  File "./dotplot.py", line 85, in <module>
    main()
  File "./dotplot.py", line 58, in main
    main_window = MainWindow(args)
  File "gui.py", line 29, in __init__
    self.new_plot()
  File "gui.py", line 81, in new_plot
    self.display_plot(dotplot)
  File "gui.py", line 295, in display_plot
    dotplot.draw(self.canvas.main_plot, self.sequences)
  File "dotplot.py", line 22, in draw
    return self.drawer.draw(plot, *args, **kwargs)
  File "drawer.py", line 90, in make_matplotlib
    elif self.show_sequence.isdigit():
AttributeError: 'int' object has no attribute 'isdigit'
```

In my opinion the problem is that default value `100` is set in `argument_parser.py` as `int` and in other place it is expected to be a string (to have `isdigit` method comes from `str` type).

File: drawer.py [hunk 1]
```diff
@@ -73,12 +75,25 @@ def make_matplotlib(self, dot_matrix, subplot, sequences):
...
+
+        if self.show_sequence == 'True':
             subplot.yaxis.set_major_locator(ticker.MultipleLocator(1))
             subplot.set_yticklabels('a' + sequences[0].sequence)
-        if len(sequences[1].sequence) < 100:
             subplot.xaxis.set_major_locator(ticker.MultipleLocator(1))
             subplot.set_xticklabels('a' + sequences[1].sequence)
+        elif self.show_sequence == 'False':
+            pass
+        elif self.show_sequence.isdigit():
```
Comment (kinga322) [inline drawer.py]: Yes, this is a good idea. Sorry I didn't check that before making a pull request.

**Comment by krassowski** [comment]
Looks good! Please, try to add some tests next time :)
I will merge this PR now; thank you for the good work.

**Pull request merged**